SimTKcommon/include/SimTKcommon/internal/Pathname.h

```diff
@@ -87,6 +87,26 @@ namespace SimTK {
  */
 class SimTK_SimTKCOMMON_EXPORT Pathname {
 public:
+    /// Given a specified working directory (swd) and path, this function
+    /// evaluates the absolute path of a given path relative to the swd and
+    /// returns the directory, fileName, and extension of the canonicalized absolute
+    /// path. This means that instead of evaluating "." as the current working
+    /// directory (cwd), the swd is used. In general, some rules are as follows
+    /// 1) If an absolute path is given by path, swd is not used. An absolute path
+    ///    is defined as a a root-relative path name and on Windows it will begin 
```

Comment:
Yeah this is why the wording here has to be careful. On Windows, if the path is root-relative with a drive letter, it will ignore swd. If it the path does not have a drive letter, it will then look at the swd for the drive letter, to resolve ambiguities (rule 3). How to clearly state this is tricky, since we expect different behaviors on a Windows machine vs. not-Windows.